qemu/tests/driver_in_use.py

```diff
@@ -56,6 +56,7 @@ def run_bg_test_simu(bg_stress_test):
         # Need to set bg_stress_run_flag in some cases to make sure all
         # necessary steps are active
         env[bg_stress_run_flag] = False
+        env["memory_stable"] = False
```

Comment:
Seems we do not need this line, can you please double check it?

Reply:
Yes, it's not a must.
Modified.
Thanks